src/__tests__/lib/containers/entity_finder/EntityFinder.test.tsx

```diff
@@ -408,6 +408,35 @@ describe('EntityFinder tests', () => {
             type: EntityDetailsListDataConfigurationType.ENTITY_SEARCH,
             query: {
               queryTerm: queryTerms,
+              // Verify that at least one of the omitted types is excluded from the search
+              booleanQuery: expect.arrayContaining([
+                {
+                  key: 'node_type',
+                  value: 'project',
+                  not: true,
+                },
+              ]),
+            },
+          },
+        }),
+        {},
+      ),
+    )
+    await waitFor(() =>
+      expect(mockDetailsList).toBeCalledWith(
+        expect.objectContaining({
+          configuration: {
+            type: EntityDetailsListDataConfigurationType.ENTITY_SEARCH,
+            query: {
+              queryTerm: queryTerms,
+              // The lone selectable type should not have been excluded
+              booleanQuery: expect.not.arrayContaining([
```

Comment:
I'd rather not admit how long it took me to recognize the expect._not_... in the test.

Reply:
Jest matchers definite leave a lot to be desired in terms of readability, especially compared to JUnit/Mockito!